src/components/View/PuntoDiContatto/PuntoDiContattoContatti.jsx

```diff
@@ -58,11 +58,12 @@ const PuntoDiContattoContatti = ({ content }) => {
       {content?.contatti?.map((pdc, i) => (
         <div key={i}>
           <div className="mt-2 d-flex">
-            <h5 className="h6 text-capitalize">
+            <h3 className="h6 text-capitalize">
```

Comment:
https://iosanita-staging.c3p0.redturtle.it/lista-blocchi/lista-ct/punto-di-contatto#:~:text=Telefono%3A,Contatto